libglusterfs/src/common-utils.c

```diff
@@ -50,7 +50,11 @@
 #include "glusterfs/syscall.h"
 #include "glusterfs/globals.h"
 #define XXH_INLINE_ALL
+#if defined(HAVE_LIBXXHASH)
+#include <xxhash.h>
+#else /* use from contrib */
```

Comment:
The #include "xxhash.h" directive is based on #define XXH_INLINE_ALL  and has nothing to do with using xxhashlib from contrib/ or from the installed library.

Ref: https://github.com/Cyan4973/xxHash#build-modifiers  and xxhash.h

80   * It also keeps xxHash symbols private to the unit, so they are not exported.
    81   *
    82   * Usage:
    83   *     #define XXH_INLINE_ALL
    84   *     #include "xxhash.h"
    85   *
    86   * Do not compile and link xxhash.o as a separate object, as it is not useful.
    87   */

Reply:
So you can leave out `if defined(HAVE_LIBXXHASH)` block and just `#include "xxhash" `

Reply:
OK